RetailCoder.VBE/Refactorings/ExtractMethod/ExtractedMethod.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Rubberduck.Parsing.Grammar;
+using Rubberduck.Parsing.Symbols;
+
+namespace Rubberduck.Refactorings.ExtractMethod
+{
+    public class ExtractedMethod : IExtractedMethod
+    {
+        public string MethodName { get; set; }
+        public Accessibility Accessibility { get; set; }
+        public bool SetReturnValue { get; set; }
+        public ExtractedParameter ReturnValue { get; set; }
+        public IEnumerable<ExtractedParameter> Parameters { get; set; }
+        public string AsString()
+        {
+            string result;
+            var argsList = string.Join(", ", Parameters.Select(p => p.Name));
+            result = "Call " + MethodName + "( " + argsList + " )";
```

Comment:
Rubberduck actively discourages the use of the `Call` keyword, which is optional and deemed obsolete.

The inspection can be disabled by the user, but I see no reason for Rubberduck to *generate* new code that uses a keyword that is otherwise hunted down.

Perhaps this (i.e. whether or not to generate `Call` statements in generated code) could become an option in the inspection settings?